packages/plexus/src/types/index.tsx

```diff
@@ -39,6 +39,7 @@ export type TEdge<T = {}> = T & {
   from: TVertexKey;
   to: TVertexKey;
   isBidirectional?: boolean;
+  edgeLabel?: string;
```

Comment:
the prefix `edge` is redundant since the type is `TEdge`

Reply:
Sure, will make it just `edge`